tests/ci/profiles/tf_hcp_cluster_profiles.yml

```diff
@@ -208,4 +208,32 @@ profiles:
     oidc_config: "managed" # id:72493
     admin_enabled: false # Not yet supported
     unified_acc_role_path: ""
-    ingress_listening_method: "internal" # id:72516
\ No newline at end of file
+    ingress_listening_method: "internal" # id:72516
+
+- as: rosa-hcp-arm # id:75107
```

Comment:
I have the same thinking in https://github.com/openshift/rosa/pull/2268 about whether we could use the existing profile and a new prow CI job to achieve the testing aim.

Reply:
Removed the profile. Added the COMPUTE_MACHINE_TYPE env configuration